packages/koa-shopify-auth/src/auth/validate-hmac.ts

```diff
@@ -11,7 +11,7 @@ export default function validateHmac(
   const {hmac: _hmac, signature: _signature, ...map} = query;
 
   const orderedMap = Object.keys(map)
-    .sort()
+    .sort((value1, value2) => value1.localeCompare(value2))
```

Comment:
I ended up disabling the rule that enforced this, but decided to leave it because it is a good practise.

Reply:
https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/require-array-sort-compare.md